pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-configuring-repository.adoc

```diff
@@ -1,11 +1,14 @@
-include::../attributes.adoc[]
-
 ifdef::context[:parent-context: {context}]
 
+include::../attributes.adoc[]
+
 [id="configuring_repository_{context}"]
 = Configuring your repository
 
+:context: assembly-configuring-repository
+
 [role="_abstract"]
+
```

Comment:
I'm not entirely sure but I think this blank line might introduce an issue in extracting the abstract here.

Reply:
Oh, yes, thanks for catching this. I'll remove the blank line.